pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -365,7 +365,7 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{Requeue: true}, nil
 	}
 
-	if r.conditions == nil {
+	if r.conditions == nil || r.conditions.empty() {
```

Comment:
I'm still a bit confused about this if.
Are we considering the unknown status when, for any reason, we fail to properly fetch conditions from all the operators?